pkg/coderefs/coderefs.go

```diff
@@ -92,6 +92,8 @@ func Scan() {
 		log.Error.Fatalf(err.Error())
 	}
 
+	deleteStaleBranches(ldApi, cmd, repoParams.Name)
```

Comment:
An alternative to consider - don't swallow all errors inside `deleteStaleBranches`, and move it to the bottom of this function. I think swallowing all errors could lead to us missing issues down the line. If deletion is the last thing that happens, coderefs will still be posted, but users will also notice failures and report issues.